Assets/Scripts/CutScene Transitions/ReturnToMenu.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class ReturnToMenu : MonoBehaviour
```

Comment:
I haven't looked to see how this is used in-game yet, but I wrote a script that looks like it does pretty much the same thing but is used by buttons. Is this not applicable in this situation?
https://github.com/MelliferaDev/mellifera/blob/aa7f5bfae41bd4bd212abe0d7101a43531f230a4/Assets/Scripts/Menus/QuitToMenu.cs

Reply:
I just wrote this script and set it up to match how the intro cut scene works, I didn't really check the other existing scripts